crates/core/src/surfnet/svm.rs

```diff
@@ -66,6 +67,7 @@ pub struct SurfnetSvm {
     pub simnet_events_tx: Sender<SimnetEvent>,
     pub geyser_events_tx: Sender<GeyserEvent>,
     pub signature_subscriptions: HashMap<Signature, Vec<SignatureSubscriptionData>>,
+    pub account_subscriptions: HashMap<Pubkey, Vec<(Option<UiAccountEncoding>, Sender<Account>)>>,
```

Comment:
From what I can tell, you don't actually end up using this `UiAccountEncoding`, do you? We can remove from this map if so